kovri.md

```diff
@@ -46,7 +46,19 @@ Let's next go over some fairly technical stuff to have a better understanding of
 
 ## 5) Kovri and Technical Attributes
 
-> **TODO**: Give a concise overview of garlic routing/encryption
+Now hold on, since we are about to overview how Kovri works.
+
+Kovri is currently built from I2P's open standards. I2P stands for the invisible internet protocol. Kovri acts as an anonymizing router as it tunnels traffic through the I2P network.
+
+To protect their anonymity, each participant connects to other network members and uses them to pass messages.
+In most circumstances, people will choose four peers: two for incoming connections, and two for outgoing connections.
+People can use more for greater privacy at the cost of performance.
+
+Suppose Dave's mining pool has just solved a block and would like to send this transaction information to the other network members.
+Dave's router will send the message to his two outgoing peers, who will then search for the incoming peers of other nodes on the network.
+Dave can commincate securely without other Monero nodes on the network knowing what his actual IP address is.
+The other participants in the I2P network do not know what their place is in the chain of routing, nor do they know what information is being relayed inside the network.
+This is called garlic routing, wherein the only information shown to peers is the instruction set for sending to the next peer.
```

Comment:
>The other participants in the I2P network do not know what their place is in the chain of routing, nor do they know what information is being relayed inside the network.
>This is called garlic routing, wherein the only information shown to peers is the instruction set for sending to the next peer.

I like the wording but this also describes onion routing (not quite garlicy). I also wonder if this should even be included (again, the peer selection isn't entirely accurate).